firebase-inappmessaging/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=19.0.4
+version=19.1.0
```

Comment:
For the next release we would need it to be a minor version bump since we have public api changes. Just so that we dont forget. Whenever the apiInformation task fails.. It suggests what kind of version update is needed.. Usually we do it as release operators since we know at the time of the release..